raster/src/main/scala/geotrellis/raster/equalization/MultibandEqualizationMethods.scala

```diff
@@ -0,0 +1,11 @@
+package geotrellis.raster.equalization
+
+import geotrellis.raster.histogram.StreamingHistogram
+import geotrellis.raster.MultibandTile
+import geotrellis.util.MethodExtensions
+
+
+trait MultibandEqualizationMethods extends MethodExtensions[MultibandTile] {
+  def equalize(histograms: Array[StreamingHistogram]): MultibandTile = HistogramEqualization(self, histograms)
```

Comment:
It currently relies on the ability to get the CDF associated with the histogram directly form it.  I could add that capability to the other histogram types, but only `StreamingHistogram` has it right now.